detox/src/artifacts/ArtifactsManager.js

```diff
@@ -134,10 +139,6 @@ class ArtifactsManager {
     await this._callPlugins('descending', 'onSuiteEnd', suite);
   }
 
-  async onInit() {
```

Comment:
No more `onInit`, because no plugin was actually using it, :open_mouth: mea culpa :blush: